eclair-core/src/test/scala/fr/acinq/eclair/payment/relay/NodeRelayerSpec.scala

```diff
@@ -584,7 +585,7 @@ class NodeRelayerSpec extends ScalaTestWithActorTestKit(ConfigFactory.load("appl
     assert(outgoingPayment.targetExpiry == outgoingExpiry)
     assert(outgoingPayment.targetNodeId == outgoingNodeId)
     assert(outgoingPayment.additionalTlvs == Nil)
-    assert(outgoingPayment.assistedRoutes == hints)
+    assert(outgoingPayment.extraEdges == Bolt11Invoice.toExtraEdges(hints, outgoingNodeId).map(GraphEdge(_)))
```

Comment:
Similar comment as the test in `EclairImplSpec`, you should test the actual contents of `extraEdges` instead of running the same code as the code under test.